src/js/components/Chart/__tests__/__snapshots__/Chart-test.js.snap

```diff
@@ -1367,11 +1601,8 @@ exports[`Chart type 1`] = `
         <title>
           zero
         </title>
-        <rect
-          height={24}
-          width={24}
-          x={-12}
-          y={180}
+        <path
```

Comment:
I tried to use the following code snippet to compare the visualization for backward compatibilty purposes:
```
<svg viewBox="0 0 220 100" xmlns="http://www.w3.org/2000/svg">
  <!-- Simple rectangle test renders as expected -->
  <rect width="24" height="24" />

  <!-- 2.14.0 version for rendering point -->
  <rect height={24} width={24} x={-12} y={180} />
  <!-- this PR replaces rect with path -->
  <path d="M -12 180 L 12 180 L 12 204 L -12 204 Z"/>
</svg>
```
But nothing was rendered for me on either test, I was hoping you'll have insights into what I'm doing wrong.
Also, I'm calling out that users that may have specific styles for `<rect />` would loose their styles due to this change.

Reply:
I think the key issue is that you can't use `{24}` in vanilla SVG. I did this to illustrate:
```
<svg width="220" height="100" viewBox="0 0 220 100" xmlns="http://www.w3.org/2000/svg">
  <rect width="24" height="24" fill="green" />
  <rect height="24" width="24" x="24" y="48" fill="blue" />
  <path d="M 48 24 L 72 24 L 72 48 L 48 48 Z" fill="red" />
</svg>
```

Reply:
I'm not concerned about the internal being `rect` as we don't guarantee DOM element consistency of internal elements.

Reply:
Agreed. I'm also guessing that Chromatic would have alerted us in case there was a visual impact on the svg when `<rect />` got replaced with `<path />`

Reply:
correct